casper-private-auction-installer/Cargo.toml

```diff
@@ -7,13 +7,14 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-casper-contract = "1.4.3"
-casper-types = "1.4.6"
-casper-private-auction-core = { path = "../casper-private-auction-core"}
+casper-contract = "1.4.4"
+casper-types = "1.5"
+casper-private-auction-core = { path = "../casper-private-auction-core" }
+base16 = { version = "0.2", default-features = false, features = ["alloc"] }
```

Comment:
If my reading of CEP-78 source is correct, we won't need this

Reply:
Removed thanks

https://github.com/CasperLabs/casper-private-auction/pull/23/commits/f21b0c5dc4a8571b7cb10c7eeafba11de701ba5b